lambdas/select-cri/src/main/java/uk/gov/di/ipv/core/selectcri/SelectCriHandler.java

```diff
@@ -46,6 +47,8 @@
     private static final String JOURNEY_FAIL = "/journey/fail";
     private static final String DCMAW_SUCCESS_PAGE = "dcmaw-success";
     private static final String APP_JOURNEY_USER_ID_PREFIX = "urn:uuid:app-journey-user-";
+    public static final String CRI_UK_DRIVING_LICENCE = "ukDrivingLicence";
+    public static final String CRI_UK_PASSPORT_AND_DRIVING_LICENCE = "ukPassportAndDrivingLicence";
```

Comment:
When the stub is enabled this will need to be `stubUkPassportAndDrivingLicence`? Normally this would come from the configured CRI id, but obviously we don't have that for passport+driving licence

Reply:
We've addressed this on the mob call. It now checks to see if the Driving License is a stub, and returns the appropriate mult-doc-check page.